streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -616,6 +616,27 @@ class StreamingContext private[streaming] (
     }
   }
 
+  /**
+   * Registers streaming listeners specified in spark.streaming.extraListeners.
+   */
+  private def setupExtraStreamingListener(): Unit = {
```

Comment:
I guess `registerXXX` is more seamless than `setupXXX` because 
- we have `registerProgressListener` already
- function description also says `Registers`.

Reply:
Done